justfile

```
@@ -110,6 +110,12 @@ download-assets *args:
     {{ if env_var_or_default("CI", "false") == "false" { "export AWS_PROFILE=\"S3BucketsDeveloperAccess-690046978283\"; echo AWS_PROFILE=$AWS_PROFILE;" } else { "" } }}
     ./download_assets.sh {{args}}
 
+upload-assets *args:
+    #!/usr/bin/env -S bash -eu -o pipefail
+    cd {{justfile_directory()}}/.github/scripts
+    {{ if env_var_or_default("CI", "false") == "false" { "export AWS_PROFILE=\"S3BucketsDeveloperAccess-690046978283\"; echo AWS_PROFILE=$AWS_PROFILE;" } else { "" } }}
+    ./prepare_data.sh {{args}}
```

❓ should we rather make the repo root the current dir? otherwise one has to set the uploadable dir `{{args}}` relative to the scripts dir.
```suggestion
    {{ if env_var_or_default("CI", "false") == "false" { "export AWS_PROFILE=\"S3BucketsDeveloperAccess-690046978283\"; echo AWS_PROFILE=$AWS_PROFILE;" } else { "" } }}
    ./.github/scripts/prepare_data.sh {{args}}
```

---

Yeah makes sense! 👍🏻